game_eggs/gta/fivem/egg-five-m.json

```diff
@@ -31,7 +31,7 @@
         {
             "name": "fivem license",
             "description": "Required to start the service. Get your keys at https:\/\/keymaster.fivem.net\/",
-            "env_variable": "FIVEM_LICENSE",
+            "env_variable": "sv_licenseKey",
```

Comment:
This requires to be `FIVEM_LICENSE`
If you wanna rename this variable here, you need to change the "files"-part where the license is being replaced in the server config AND remove the unset-command